src/electionguardtest/ballot_factory.py

```diff
@@ -28,7 +29,7 @@
 _T = TypeVar("_T")
 _DrawType = Callable[[SearchStrategy[_T]], _T]
 
-here = os.path.abspath(os.path.dirname(__file__))
+data = os.path.realpath(os.path.join(__file__, "../../../data"))
```

Comment:
This will fail if this is packaged right? Are we still treating `electionguardtest` as a package? There is no piece of it in the setup.py...